### ISSUE TITLE
Andrew Karr Feature Request

### ISSUE THREAD

**Issue by ryantimjohn** [ryantimjohn/action_network_1on1_tracker#2]
I think it’d be a 💯 feature if  you could use the check in feature for actions/events to notify a organizer/activist when someone they’re organizing has attended something. Even better to get monthly recaps of actions that summarized how you did. Seems reasonable, no?

**Comment by smithsa** [comment]
We can do monthly reports and send out notifications via an email, record the data somewhere so they can pull it up again?

I also like the idea of having a small dashboard for something like this. But it would work better performance wise to store the data of people,events, etc ourselves. We would need to create a script to backfill the data. Sounds like a nice to have down the line though.

**Comment by ryantimjohn** [comment]
Yeah, definitely thinking of this as a down the line thing. If we just had a user-friendly, quick-to-enter one-on-one tracker, I think everyone would be thrilled.
